<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="WrapButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wrap and Upload</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>source dir :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_source_folder">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>871</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>not set</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>831</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>check file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>upload URL:</string>
    </property>
   </widget>
   <widget class="QLabel" name="url_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>510</y>
      <width>851</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>not set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>server credentials:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_server_credentials">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>not set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>separator string:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_separator">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>not set</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1238</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="actionset_upload_path">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>set upload path</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
